_posts/2021-03-17-cockpit-240.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 240
+author: mvo
+date: 2021-03-17
+tags: cockpit linux
+slug: cockpit-240
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 240.
+
+## New localization: Norwegian Bokmål
+
+Cockpit is now also available in Norwegian Bokmål localization. Thank
+you Stein-Aksel Basma for this contribution!  Cockpit is being
+translated in [Weblate](https://translate.fedoraproject.org/projects/cockpit/master/)
+where anyone can help to improve Cockpit.
+
+## Performance metrics: Journal integration
+
+For every event on metrics page relevant journal logs are provided to
+help identify why some event has happened.
+
+![Journal entries for a spike](/images/metrics-journal.png)
```

Comment:
Without looking at the image, it's not at all clear what "for a spike" means.

Reply:
Yes, should be "for an event", to refer back to the text.